resources/lib/tubecast/youtube/app.py

```diff
@@ -241,7 +241,10 @@ def handle_cmd(self, cmd):
                 self.cur_list = video_ids.split(",")
                 # Set info on our custom player instance and request playback
                 self.player.setInfo(cur_video_id, self.ctt, self.cur_list_id, self.current_index)
-                play_url = kodibrigde.get_youtube_plugin_path(cur_video_id, seek=data.get("currentTime"))
+                current_time = data.get("currentTime", "0")
+                if not current_time.isdigit():
+                    current_time = "0"
+                play_url = kodibrigde.get_youtube_plugin_path(cur_video_id, seek=current_time)
```

Comment:
I'd use:

```python
current_time = data.get("currentTime", 0)
play_url = kodibrigde.get_youtube_plugin_path(cur_video_id, seek=current_time)
```

and

```python 
def get_youtube_plugin_path(videoid, seek=0):  
    return "plugin://plugin.video.youtube/play/?video_id={}&seek={}".format(videoid, seek)
```

Reply:
I agree. I thought you wanted to have the type validation in there. I'm sorry if I misunderstood what you meant.

Anyway, I changed it and I'm sorry for making this more difficult than it should've been.